### PR TITLE
Feat/mzad 29/reusable album card component

### PR DESCRIPTION
**Pull request by GvantsaBarnabishvili**

### CODE REVIEW

File: app/Components/Dropdown/Dropdown.tsx [hunk 1]
```diff
@@ -1,27 +1,52 @@
...
 const Dropdown: DropdownType = (props: DropdownPropsInterface) => {
   const [show, setShow] = useState<boolean>(false);
+  const dropdownRef: RefObject<HTMLDivElement> = useRef<HTMLDivElement>(null);
 
   const onClick = (): void => {
     setShow(!show);
   };
 
+  useEffect(() => {
+    // Function to close dropdown when clicking outside
```
Comment (pailodze) [inline app/Components/Dropdown/Dropdown.tsx:17]: Refactor This

File: app/Components/AlbumCard/AlbumCard.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+import { DropDownItemsInterface } from '../Dropdown/interfaces/dropdown-items-props.interface';
+import Icon from '../Icon/Icon';
+import { IconNameEnum } from '../Icon/enums/icon-name.enum';
+import { TextHtmlTypeEnum } from '../Text/enums/text-html-type.enum';
+import { TextTypeEnum } from '../Text/enums/text-type.enum';
+import styles from './AlbumCard.module.scss';
+import { AlbumCardPropsInterface } from './interfaces/album-card-props.interface';
+import { AlbumCardType } from './types/albumcard.type';
+import Text from '@/app/Components/Text/Text';
+const AlbumCard: AlbumCardType = (props: AlbumCardPropsInterface) => {
```
[the remark this answers is not in the document]
Comment (GvantsaBarnabishvili) [reply app/Components/AlbumCard/AlbumCard.tsx:13]: which Items? please be more specific

File: app/Components/Dropdown/Dropdown.tsx (hunk 1 above)
Comment (GvantsaBarnabishvili) [reply app/Components/Dropdown/Dropdown.tsx:17]: I will do that

File: app/Components/AlbumCard/AlbumCard.module.scss [hunk 3]
```diff
@@ -0,0 +1,45 @@
+@import '../../Colors/common-colors.scss';
+@import '../../Colors/dark-colors.scss';
+@import '../../Colors/light-colors.scss';
+.AlbumCard{
```
Comment (pailodze) [inline app/Components/AlbumCard/AlbumCard.module.scss:4]: Classnames should start in lowerCase

Comment (pailodze) [reply app/Components/AlbumCard/AlbumCard.module.scss:4]: use camelCase Not pascal

File: app/Components/Dropdown/Dropdown.tsx [hunk 4]
```diff
@@ -1,27 +1,50 @@
...
+      }
+    };
+    document.addEventListener('mousedown', handleClickOutside);
+
+    return (): void => {
+      document.removeEventListener('mousedown', handleClickOutside);
+    };
+  }, []);
+
+  console.log(props.items, 'e3gfio0jegrwoiperfgk');
```
Comment (pailodze) [inline app/Components/Dropdown/Dropdown.tsx:36]: Remove this console log

File: app/Components/Dropdown/Dropdown.tsx [hunk 5]
```diff
@@ -1,27 +1,50 @@
...
   const [show, setShow] = useState<boolean>(false);
+  const dropdownRef: RefObject<HTMLDivElement> = useRef<HTMLDivElement>(null);
 
   const onClick = (): void => {
     setShow(!show);
   };
 
+  useEffect(() => {
+    const handleClickOutside = (event: MouseEvent): void => {
+      console.log(
```
Comment (pailodze) [inline app/Components/Dropdown/Dropdown.tsx:18]: remove console log

File: app/Components/Dropdown/Dropdown.tsx [hunk 6]
```diff
@@ -1,27 +1,50 @@
...
+        dropdownRef.current,
+        dropdownRef.current?.contains(event.target as Node),
```
Comment (pailodze) [inline app/Components/Dropdown/Dropdown.tsx:20]: Code is not readable

File: app/page.tsx [hunk 7]
```diff
@@ -1,20 +1,28 @@
...
   return (
     <div>
-      <Button
-        type={ButtonTypeEnum.Primary}
-        onClick={() => console.log('darxeuli maq')}
-      >
-        uihi8yughiy8
-      </Button>
-      <Input type={InputTypeEnum.Text} placeholder={'Search'} isDark={false} />
+      <AlbumCards
```
Comment (pailodze) [inline app/page.tsx:7]: This props are useless

**Review by pailodze — CHANGES_REQUESTED** [review]

**Pull request merged**